src/components/Board.jsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components'
 
 const Board = ({ board }) => (
   <BoardCard color={board.color}>
-    <h3>{board.number}e bestuur</h3>
+    <h3><a href={'/besturen/' + board.slug.replace(/\W+/g, '-').toLowerCase()}>{board.number}e bestuur</a></h3>
```

Comment:
The regex replace is unnecessary in combination with slug, as slug is being edited to be this compliant in Contentful by default :)

Reply:
will remove it :)